app/waf/src/controllers/adc.controller.ts

```diff
@@ -153,6 +147,11 @@ export class AdcController extends BaseController {
           await runWithTimer(`${adc.id}.onboard`, () =>
             this.onboard(adc, addonReq),
           );
+
```

Comment:
The logic is for the Hardware bigip?

Reply:
No logic for HW, because we postpone HW support.